tuxemon/core/prepare.py

```diff
@@ -34,52 +34,48 @@
 
 import logging
 import os
+import os.path
 
 import pygame as pg
 
 from .components import config
-from .platform import get_config_path
+from tuxemon.constants import paths
 
 logger = logging.getLogger(__name__)
 
-# Get the tuxemon base directory
-BASEDIR = os.path.abspath(os.path.join(os.path.dirname(os.path.realpath(__file__)), "..")) + os.sep
-if "library.zip" in BASEDIR:
-    BASEDIR = os.path.abspath(os.path.join(BASEDIR, "..")) + os.sep
 
-# Set up our config directory
-CONFIG_PATH = get_config_path() + "/.tuxemon/"
+# Create game dir if missing
 try:
-    os.makedirs(CONFIG_PATH)
+    os.makedirs(paths.USER_GAME_DIR, exist_ok=True)
```

Comment:
We don't need "exist_ok", all it does it break python2. 
The `except OSError` is enough.

Reply:
Please check that this works for PR works for py2 as well.